### PR TITLE
media-libs/nv-codec-headers-9.1.23.1: version bump and  media-video/ffmpeg-9999: fix nv-codec-headers dep 

### PR DESCRIPTION
**Pull request by fedeliallalinea**
media-video/ffmpeg-9999[cuda] need media-libs/nv-codec-headers-9.1.23.1 for compile.

### CODE REVIEW

File: media-libs/nv-codec-headers/nv-codec-headers-9.1.23.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+SRC_URI="https://github.com/FFmpeg/nv-codec-headers/releases/download/n${PV}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	>=x11-drivers/nvidia-drivers-390.25[${MULTILIB_USEDEP}]
```
Comment (aballier) [inline media-libs/nv-codec-headers/nv-codec-headers-9.1.23.1.ebuild]: Please consider updating this to match the version in their README

**Comment by fedeliallalinea** [comment]
Sorry for delay I've not see email

**Comment by aballier** [comment]
Seems we need to drop x86 keywords there

**Comment by ajakk** [comment]
All changes here are already in tree at this point. Thank you for your effort, in any case!

**Pull request closed without merging**